static/src/js/components/AccessMethod/SwodlrForm.jsx

```diff
@@ -0,0 +1,439 @@
+                  type="radio"
+                  id="granule-extent-256-by-128"
+                  checked={granuleExtent}
+                  onChange={
+                    () => {
+                      handleGranuleExtent(true)
+                    }
+                  }
+                />
+              </div>
+            </Form>
+          </Col>
+        </Row>
+        <Row>
+          <Col>
+            Sampling Grid Type
+            <OverlayTrigger
+              placement="top"
+              overlay={
+                (
+                  <Tooltip style={{ width: '20rem' }}>
+                    {swodlrToolTips.SamplingGridResolution}
+                  </Tooltip>
+                )
+              }
+            >
+              <EDSCIcon icon={FaQuestionCircle} size="16px" variant="details-span" />
+            </OverlayTrigger>
+          </Col>
+          <Col>
+            <Form>
+              <div className="mb-3">
+                <Form.Check
+                  inline
+                  label="UTM"
+                  name="sample-grid"
+                  type="radio"
+                  id="sample-grid-utm"
+                  checked={sampleGrid === 'UTM'}
+                  onChange={
+                    () => {
+                      handleSampleGrid('UTM')
+                    }
+                  }
+                />
+                <Form.Check
+                  inline
+                  label="LAT/LON"
+                  name="sample-grid"
+                  type="radio"
+                  id="sample-grid-lat-lon"
+                  checked={sampleGrid === 'GEO'}
+                  onChange={
+                    () => {
+                      handleSampleGrid('GEO')
+                    }
+                  }
+                />
+              </div>
+            </Form>
+          </Col>
+        </Row>
+        <Row>
+          <Col>
+            Raster Resolution
+            <OverlayTrigger
+              placement="top"
+              overlay={
+                (
+                  <Tooltip style={{ width: '20rem' }}>
+                    {swodlrToolTips.RasterResolution}
+                  </Tooltip>
+                )
+              }
+            >
+              <EDSCIcon icon={FaQuestionCircle} size="16px" variant="details-span" />
+            </OverlayTrigger>
+          </Col>
+          <Col>
+            <Form>
+              <Form.Control
+                as="select"
+                onChange={
+                  (e) => {
+                    handleRasterResolutionUpdate(e)
+                  }
+                }
+                data-testid="rasterResolutionSelection"
+                value={rasterResolution}
+              >
+                {
+                  sampleGrid === 'GEO'
+                    ? geoRasterOptions.map((option) => (
+                      <option value={option.value} key={option.value} data-testid={`geo-raster-selection-${option.value}`}>
```

Comment:
I was drawn to this because it looked like a lot of code being duplicated, you shouldn't need to define the entire option markup twice.

Why are we using a testid? We should avoid using those